wiki/Internet_Relay_Chat/fr.md

```diff
@@ -1,65 +1,75 @@
+1. Assurez-vous que vous utilisez le mot de passe donné sur la page [IRC Authentication](https://osu.ppy.sh/p/irc).
+2. Si votre nom d'utilisateur comporte des espaces, remplacez-le par des underscores. (exemple : **Ce Nom** par **Ce_Nom**)
 
-| Client                                    | Commandes                                                                                                                                                                                                             |
-|-------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| [HexChat](http://hexchat.github.io/)      | a. Faites un clic droit sur ​​le canal souhaité, puis cochez la case "Masquer les messages de connexion/déconnexion " dans le sous-menu Paramètres.  
+### Puis-je utiliser un autre nom d'utilisateur?
+Non, vous ne pouvez utiliser que le nom d'utilisateur de votre compte osu!.
 
-                                             b. Allez dans Options » Préférences, sous Discussion » Général, cochez la case "Masquer les messages de connexion/déconnexion".                                                                                        |
-| [Irssi](http://www.irssi.org)             | Tapez /ignore -channels \#nomducanal \* JOINS PARTS QUITS                                                                                                                                                             |
-| [KVIrc](http://www.kvirc.net)             | Reportez-vous à [ce lien](http://www.kvirc.ru/forum/?topic=609.0) menant aux forums officiels de KVIrc.                                                                                                               |
-| [mIRC](http://www.mirc.com/)              | Dans Outils » Options » sélectionnez "IRC". Cliquez sur le bouton "Evènements". Changez "connexions", "déconnexions" et "surnoms" à "Hide". [Plus d'informations ici](http://i.clintecker.com/disable-irc-msgs.html). |
-| [Quassel IRC](http://www.quassel-irc.org) | Faites un clic droit sur ​​la fenêtre de discussion, choisissez "Masquer les évènements" puis "Connexion/déconnexion".                                                                                                |
-| [XChat](http://www.xchat.org)             | Faites un clic droit sur le canal désiré. Dans le menu, décochez "Montrer les messages de connexion et déconnexion".                                                                                                  |
+### Quel est le statut de ma voix? Je vois aussi des gens en avoir.
+Les utilisateurs avec *statut vocal* sont également connectés via un client IRC, à l'exception des modérateurs de discussion qui ont toujours le statut *opérateur (+ o)*, quel que soit le client utilisé.
```

Comment:
```suggestion
Les utilisateurs avec *statut vocal* sont également connectés via un client IRC, à l'exception des modérateurs de discussion qui ont toujours le statut *opérateur (+o)*, quel que soit le client utilisé.
```
`+o` is an IRC marker for operators, so this shouldn't follow normal French grammar rules